### PR TITLE
Setup service monitor for prometheus and alertmanager

### LINKED ISSUE #278: [Feature] The prometheus-operator should scrape prometheus and alertmanager metrics by default
or at least in the `contrib/kube-prometheus` example.

@fabxc @brancz

### PR DESCRIPTION
**Pull request by gianrubio**
Fix #278

### CODE REVIEW

File: Documentation/user-guides/cluster-monitoring.md [hunk 1]
```diff
@@ -361,6 +361,32 @@ spec:
     interval: 15s
   - port: http-metrics-skydns
     interval: 15s
+---
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: ServiceMonitor
+metadata:
+  name: prometheus
+  labels:
+    k8s-apps: prometheus
```
Comment (sdurrheimer) [inline Documentation/user-guides/cluster-monitoring.md]: Shouldn't this be `prometheus: k8s` to match other prometheus components ? I think `k8s-apps` is reserved to `kubernetes` components like the `apiserver` or the `kube-controller`.

Comment (gianrubio) [inline Documentation/user-guides/cluster-monitoring.md]: You're right, fixed

**Comment by mxinden** [comment]
Test this please

Just to check the assets pipeline.

**Comment by mxinden** [comment]
Test this please

**Comment by mxinden** [comment]
Test this please

**Comment by mxinden** [comment]
test this please

**Comment by gianrubio** [comment]
@mxinden I already tested this, it's the same as I have in my  cluster

**Comment by mxinden** [comment]
@gianrubio Sorry for that, tried to trigger our Jenkins CI, but somehow the hook seems to be broken. 

LGTM. If no other concerns I will merge it.

**Comment by gianrubio** [comment]
> @gianrubio Sorry for that, tried to trigger our Jenkins CI, but somehow the hook seems to be broken.

So let's wait the test pass, so I'll not break other developers :)

**Comment by mxinden** [comment]
test this please

**Comment by mxinden** [comment]
@gianrubio Thanks for the contribution! Sorry for the trouble and the notification spam.

**Pull request merged**